documentation/asciidoc/computers/compute-module/cmio-display.adoc

```diff
@@ -61,3 +61,21 @@ To entirely ignore the display when connected, add the following line to `/boot/
 ----
 ignore_lcd=1
 ----
+
+== Attaching the Touch Display 2 LCD panel
+
+Touch Display 2 is a 720x1280 7" LCD display designed specifically for Raspberry Pi devices (see https://www.raspberrypi.com/products/touch-display-2/). It connects in much the same way as the original touch display, but the software setup on Compute Modules is slightly different as it uses a different display driver. See xref:../accessories/touch-display-2.adoc[Touch Display 2] for connection details.
+
+Edit the /boot/firmware/config.txt file and add the following to enable Touch Display 2 on DSI1.
```

Comment:
I've just noticed that the original text on this page talks about DISP0 / DISP1, whereas this new text talks about DSI0 / DSI1 ? :thinking:

Reply:
I've replaced all occurrences with DISP0/DSI0  etc. I think it looks OK like that. It's a weird one as the board silkscreen says DISP0, but the software all uses DSI0.